cmd/service_bootstrap_desktop.go

```diff
@@ -134,7 +134,7 @@ func (di *Dependencies) bootstrapServiceOpenvpn(nodeOptions node.Options) {
 		proposal := openvpn_discovery.NewServiceProposalWithLocation(currentLocation, transportOptions.Protocol)
 
 		var portSupplier port.ServicePortSupplier = di.PortPool
-		if transportOptions.Port != 0 {
+		if transportOptions.Port != 0 && locationInfo.PubIP == locationInfo.OutIP {
```

Comment:
There was a problem with a static port configuration that @tadaskay discovered.
It was removed to fix the problem. Is it ok to return it back?  
https://github.com/mysteriumnetwork/node/pull/896